sites/all/themes/orwell/sass/component/_spotlight-box.scss

```diff
@@ -5,11 +5,16 @@
   margin: -20px;
 }
 
-.spotlight-box__top__primary,
-.spotlight-box__top__secondary,
-.spotlight-box__middle__item,
-.spotlight-box__bottom__item {
-  padding: 20px;
+// Rather than setting the padding of the item, set the margin of the
+// contained element(s). IE always uses box-sizing: content-box for
+// flex items, causing it to miscalculate the widths when there's a
+// padding/border. Adding the needed space as margin to the contained
+// element makes it able to calculate the proper width.
+.spotlight-box__top__primary > *,
+.spotlight-box__top__secondary > *,
+.spotlight-box__middle__item > *,
+.spotlight-box__bottom__item > * {
```

Comment:
It seems kinda extreme to use * wildcard - can you tell me why its necessary? :)

Reply:
Because, basically those elements can contain anything? `div` would be a safe bet, but just in case...

Reply:
but do you still want it to have the margin property if its not a div?

Reply:
Most likely, yes.